redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/UIViewFlexContainer.kt

```diff
@@ -31,52 +26,51 @@ import app.cash.redwood.ui.Default
 import app.cash.redwood.ui.Density
 import app.cash.redwood.ui.Margin
 import app.cash.redwood.widget.UIViewChildren
-import kotlinx.cinterop.CValue
-import kotlinx.cinterop.cValue
-import kotlinx.cinterop.useContents
-import platform.CoreGraphics.CGRectMake
-import platform.CoreGraphics.CGRectZero
-import platform.CoreGraphics.CGSize
-import platform.CoreGraphics.CGSizeMake
-import platform.UIKit.UIScrollView
+import app.cash.redwood.yoga.AlignItems
+import app.cash.redwood.yoga.FlexDirection
+import app.cash.redwood.yoga.JustifyContent
 import platform.UIKit.UIView
-import platform.UIKit.UIViewNoIntrinsicMetric
 
 internal class UIViewFlexContainer(
   private val direction: FlexDirection,
 ) : Row<UIView>, Column<UIView> {
-  private val container = FlexContainer().apply {
-    flexDirection = direction
-    roundToInt = false
-  }
-  private val view = FlexContainerHostView().apply {
-    showsHorizontalScrollIndicator = false
-    showsVerticalScrollIndicator = false
-  }
-
-  override val value: UIView get() = view
+  private val yogaView = YogaUIView()
+  private val density = Density.Default
```

Comment:
Inline? Or is this going to potentially be a `var` or something later on?

Reply:
iOS will always use `Density.Default` as its coordinate system accounts for density by default. Will inline!